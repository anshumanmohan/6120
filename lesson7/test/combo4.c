int func(int a, float b)
{
  int x = 42 / a;
  float y = 42.0 / b;
  return x / y;
}

int main()
{
  int a = func(3, 3.0);
  return a;
}