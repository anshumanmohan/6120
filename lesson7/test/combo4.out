I saw a function called func!
I saw a function called main!
